documentation/source/users/rmg/liquids.rst

```diff
@@ -308,13 +299,90 @@ The work is in progress to implement the temperature dependent solvation free en
+
+where :math:`N_{\mathrm{atom}}` is the number of heavy atoms in a molecule and RSC and LDI corrections are applied for
+each ring cluster and long distance interaction group found in a molecule, respectively.
+
+While most molecules follow the equation above, halogenated molecules are treated differently in RMG for solute descriptor
+estimations: all halogen atoms are first replaced by hydrogen atoms, then the group additivity estimate is made on the
+replaced structure, and halogen corrections are lastly added for each halogen atom to get the final prediction as shown below:
+
+.. math::
+
+	E, S, A, B,\ \mathrm{or}\ L = \sum_{i=1}^{N_{\mathrm{atom*}}} \mathrm{AC}_{i} + \sum_{j=1} \mathrm{RSC}_{j} + \sum_{k=1} \mathrm{LDI}_{k} + \sum_{l=1}^{N_{\mathrm{halogen}}} \mathrm{Halogen}_{l}
+
+    = (E, S, A, B,\ \mathrm{or}\ L)_{\mathrm{replaced\ compound}} + \sum_{l=1}^{N_{\mathrm{halogen}}} \mathrm{Halogen}_{l}
+
+where the subscript 'replaced compound' denotes the compound whose halogen atoms are replaced by hydrogen atoms,
+and :math:`N_{\mathrm{atom*}}` and :math:`N_{\mathrm{halogen}}` represent the number of non-halogen heavy atoms and
+the number of halogen atoms in a molecule, respectively. We take this unique approach because it allows one to use the
+experimental solute parameter data of a replaced compound and simply apply halogen corrections to get more accurate
+estimates for a halogenated compound.
+
+For radical species, RMG uses the hydrogen bond increment (HBI) method developed by Lay et al. [Lay]_ to estimate the
+solute descriptors. The HBI method first saturates the compound, gets the estimate using the saturated structure, and
+applies the radical corrections for each radical site using the original structure as shown below:
+
+.. math::
+
+	E, S, A, B,\ \mathrm{or}\ L = (E, S, A, B,\ \mathrm{or}\ L)_{\mathrm{saturated}} + \sum_{m=1}^{N_{\mathrm{radical}}} \mathrm{radical}_{m}
+
+where the subscript 'saturated' denotes the compound whose radical sites are saturated and :math:`N_{\mathrm{radical}}`
+is the number of radical sites in a molecule. If the solute descriptors for the saturated structure are available in the
+RMG-database solute library, the group additivity approach will use the library data and apply the HBI corrections to
+get the estimates for the radical. If the solute descriptors for the saturated structure are not available in the solute
+library, RMG will use the group additivity approach to estimate the solute descriptors for the saturated structure
+and apply the HBI corrections.
+
+If a species is a radical and contains any halogen atoms, RMG's solute descriptor group additivity method will first saturate
+the compound and check whether the saturated form can be found in the solute libraries. If yes, it uses the
+solute data of the saturated from and applies the HBI corrections on the original molecule.
```

Comment:
"from" => "form"

Reply:
it's fixed in this fixup commit 2d234db.